src/openrct2/audio/Audio.cpp

```diff
@@ -311,6 +316,22 @@ void audio_stop_all_music_and_sounds()
     audio_stop_weather_sound();
 }
 
+int32_t audio_get_device_count()
+{
+    return static_cast<int32_t>(_audioDevices.size());
```

Comment:
Do we need the cast here?

Reply:
If I return size_t we would need quite a few casts else where.